static/src/stylesheets/module/experiments/_acquisitions-epic-design-variations.scss

```diff
@@ -67,7 +67,8 @@
 // ============
 
 .contributions__paragraph--highlight {
-    background-color: transparentize($multimedia-main-2, .6);
+    background-color: #ffe399;
+    padding: 4px;
```

Comment:
With my eagle DesignEye™ I think there is no left or right padding on the highlighting as it adds extra space.